Tasks/SqlAzureDacpacDeployment/DeploySqlAzure.ps1

```diff
@@ -39,6 +39,40 @@ Import-Module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
 # Load all dependent files for execution
 Import-Module ./Utility.ps1 -Force
 
+function ThrowIfMultipleFilesOrNoFilePresent($files, $pattern)
+{
+    if ($files -is [system.array])
+    {
+        throw (Get-LocalizedString -Key "Found more than one file to deploy with search pattern {0}. There can be only one." -ArgumentList $pattern)
+    }
+    else
+    {
+        if (!$files)
+        {
+            throw (Get-LocalizedString -Key "No files were found to deploy with search pattern {0}" -ArgumentList $pattern)
+        }
+    }
+}
+
+Write-Host "DacpacFilePath= Find-Files -SearchPattern $DacpacFile"
+$DacpacFilePath = Find-Files -SearchPattern $DacpacFile
+Write-Host "packageFile= $DacpacFilePath"
```

Comment:
have you seen how does it show multiple files in console. i think for multiple values we might have to use format-list. just check how it is being displayed in console window